File: QE-Selenium-Java/Activity9.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {

    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/keyboard-events");

        System.out.println("Page Title: " + driver.getTitle());

        driver.findElement(By.tagName("body")).click();

        Actions actions = new Actions(driver);
        actions.sendKeys("Hello from Selenium").perform();

        Thread.sleep(5000);

        System.out.println("Text Typed: Hello from Selenium");

        driver.quit();
    }
}

File: QE-Selenium-Python/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
import time

driver = webdriver.Firefox()

driver.get("https://training-support.net/webelements/keyboard-events")

print("Page Title:", driver.title)

driver.find_element(By.TAG_NAME, "body").click()

actions = ActionChains(driver)
actions.send_keys("Hello from Selenium").perform()

time.sleep(5)

print("Text Typed: Hello from Selenium")

driver.quit()
